### DISCUSSION TITLE
Pre-Release v2.3.8834

### DISCUSSION THREAD

**Discussion by gbakeman** [nutdotnet/WinNUT-Client#141] [Announcements]
## What's Changed
* Fixed the bug where the UPS using battery power was not displayed. by @MaxMadcc in https://github.com/nutdotnet/WinNUT-Client/pull/137
* ⚠️ Fix Application Settings not carrying over on upgrade/update in https://github.com/nutdotnet/WinNUT-Client/pull/140

## New Contributors
* @MaxMadcc made their first contribution in https://github.com/nutdotnet/WinNUT-Client/pull/137

**Full Changelog**: https://github.com/nutdotnet/WinNUT-Client/compare/v2.3.8824...v2.3.8834

<hr /><em>This discussion was created from the release <a href='https://github.com/nutdotnet/WinNUT-Client/releases/tag/v2.3.8834'>Pre-Release v2.3.8834</a>.</em>

**Comment by masterwishx** [comment]
portable file named 2.3.8824 instead of 2.3.8834 ...

**Comment by masterwishx** [comment]
Also log still in 2.3.8824 :  

![image](https://github.com/nutdotnet/WinNUT-Client/assets/28630321/1b619faa-3848-4b7f-a695-ad66a89d5f39)

**Reply by masterwishx** [reply]
> Also log still in 2.3.8824 :
> 
> ![image](https://private-user-images.githubusercontent.com/28630321/311491841-1b619faa-3848-4b7f-a695-ad66a89d5f39.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNTYyNTgsIm5iZiI6MTcxMDA1NTk1OCwicGF0aCI6Ii8yODYzMDMyMS8zMTE0OTE4NDEtMWI2MTlmYWEtMzg0OC00YjdmLWE2OTUtYWQ2NmE4OWQ1ZjM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDA3MzIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyZDBiZjUwMjk5MmM0YzNjMzk4MWJiMWRjODRmNzc2NWMzMDJjMTUwOTM5ZmRkNDllYWNjNWE4ZGU3YTVkYzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GBJ8BcqLoo2vEdWwQSfJ93dEU-hR8nDqvxf7Y_c4Gk4)

its OK , When set debug log its made right dir and log , just found new  log in old version on info log was in settings ...

**Reply by gbakeman** [reply]
It sounds like you found the log file in the correct location now? I just discovered that the log file controls are not enabled/disabling correctly, so I've created a bug to look into that (#142 ). I also fixed the name of the zip archive for the portable download, thank you. ☺️
